### ISSUE TITLE
Review and Organize Github Issue Templates

### ISSUE THREAD

**Issue by entrotech** [hackforla/tdm-calculator#1593]
### Overview
We should have a standard set of Github issue templates, so that team members are aided in creating standardized Issue formats for various types of Issues (for example Feature Requests, Bug Reports, Decisions, Business Process Analysis).

### Explanation
Right now, when you create a new Github issue, you are presented with a list of Issue Types to choose from:
![image](https://github.com/hackforla/tdm-calculator/assets/9939032/13b86446-aaf4-4a1b-8d94-371443831f7c)

Selecting the correct issue type helps the user to format the new by pre-populating the Title, Labels and Description with specific text.  This issue is for the Project and/or Product Managers to review the current list of types and the formats they define and define a better set of templates for the TDM project. Once they are defined a follow-up issue can be created for developers to implement the changes

### Action Items

- [ ] Review the issue types
- [ ] Identify obsolete and irrelevant issue types to delete.
- [ ] Review the format of issue types we want to keep and identify desired changes.
- [ ] Determine if there are additional types of issues we should have and define them

### Resources/Instructions
- [Epics for Accessibility](https://github.com/hackforla/tdm-calculator/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22feature%3A%20Accessibility%22%20label%3Aepic)

Each issue template will need this info, except assignees (issues are never preassigned)
```
---
name: 
about: ' '
title: ''
labels: 'Complexity: Missing, Feature Missing, role missing, size: missing'
assignees: ''

---
```

**Comment by ngorokh** [comment]
### List of Current Issue Templates

Using checkboxes to indicate which templates to keep/review/remove.

#### General Templates 
- [ ] [Blank Issue](https://github.com/hackforla/tdm-calculator/issues/new?template=blank-issue.md)
- [ ] [Bug report](https://github.com/hackforla/tdm-calculator/issues/new?template=bug-report.md)
- [ ] [Feature request](https://github.com/hackforla/tdm-calculator/issues/new?template=feature--request.md)
- [ ] [Development Handoff Issue](https://github.com/hackforla/tdm-calculator/issues/new?template=development-handoff-issue.md)
- [ ] [Design Issue](https://github.com/hackforla/tdm-calculator/issues/new?template=design-issue.md)


#### Content / Other 
- [ ] [Content: FAQ database: Add content](https://github.com/hackforla/tdm-calculator/issues/new?template=content--faq-database--add-content.md)
- [ ] [Content: FAQ database: Revise content](https://github.com/hackforla/tdm-calculator/issues/new?template=content--faq-database--revise-content.md)
- [ ] [Post an open role](https://github.com/hackforla/tdm-calculator/issues/new?template=post-an-open-role.md)


#### Accessibility-related (to reorganize)
- [ ] [Dev: WAVE Page Audit](https://github.com/hackforla/tdm-calculator/issues/new?template=dev--wave-page-audit.md)
- [ ] [Lighthouse: Accessibility - Forms](https://github.com/hackforla/tdm-calculator/issues/new?template=lighthouse--accessibility---forms.md)
- [ ] [Lighthouse: Accessibility - Links](https://github.com/hackforla/tdm-calculator/issues/new?template=lighthouse--accessibility---links.md)
- [ ] [Lighthouse: Cross-origin destinations are unsafe](https://github.com/hackforla/tdm-calculator/issues/new?template=lighthouse--accessibility---links.md)
- [ ] [Lighthouse: How To](https://github.com/hackforla/tdm-calculator/issues/new?template=lighthouse--how-to.md)
- [ ] [Lighthouse: Image Optimization](https://github.com/hackforla/tdm-calculator/issues/new?template=lighthouse--image-optimization.md)
- [ ] [Wave Chrome Extension: Accessibility review](https://github.com/hackforla/tdm-calculator/issues/new?template=wave-chrome-extension--accessibility-review.md)
- [ ] [Which accessibility testing tool should you use?](https://github.com/hackforla/tdm-calculator/issues/new?template=which-accessibility-testing-tool-should-you-use-.md)

**Comment by ngorokh** [comment]
### Proposal: Simplify Accessibility Templates

Currently, accessibility-related templates are split across multiple categories (WAVE, Lighthouse, and specific issue types like Forms, Links, etc.), which creates duplication and complexity.

#### Proposed approach:

Create 2 main templates:
1. Dev: Accessibility (WAVE)
2. Dev: Accessibility (Lighthouse)

#### Each template would include:
- Description
- Type of issue (Forms, Links, Images, Contrast, Other)
- Source (WAVE / Lighthouse)
- Steps to reproduce
- Suggested fix


#### Example of selecting issue source and type:
```
### Source
- [ ] WAVE
- [ ] Lighthouse

### Type of issue
- [ ] Forms
- [ ] Links
- [ ] Images
- [ ] Contrast
- [ ] ARIA
- [ ] Other

```
This would:
- reduce duplication
- simplify issue creation
- make templates easier to maintain

**Comment by ngorokh** [comment]
# Status
- [x] keep
- [ ] delete 

## Title: Dev: WAVE Page Audit: [page-name]
## Labels

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/cefcac96-3016-42a9-8768-2ee4cbef4a66" />

## Body
### Overview
We need to run the WAVE evaluation browser extension on [page-name] to audit the page for WCAG2.2 accessibility compliance. 

#### Details
We will need to audit all user interactions on the page so that all possible states and user flows are compliant.
- Page URL: [insert-page-url-here]
  - Requires sign in: [true/false]
    - User Profile Required: [None/User/Admin/Security]
  - Can access page directly from URL: [true/false]

### Action Items

- [ ] Review wiki page: Accessibility, **How to run the WAVE browser extension (Resource 1.01.02)**
- [ ] Install the WAVE browser extension
  - (suggested) pin to your browser's toolbar for quick access
- [ ] Open TDM WAVE Accessibility Audit Spreadsheet, **1. Page Log (Resource 1.02.01)**
  - [ ] Choose the row that has your issue number and populate the following fields...
    - [ ] Populate `Auditor` cell with your GitHub handle
    - [ ] Populate `Begin Date` with today's date (YYYY-MM-DD)
  - [ ] Log in as the user indicated under Details and navigate to the page you will audit
    - [ ] Run the WAVE extension with no elements focused, or clicked
      - [ ] In TDM WAVE Accessibility Audit Spreadsheet, **2. Audit Log (Resource 1.02.02)**, create a row for every Error, Contrast Error, and Alert on the Details pane
        - [ ] populate `Auditor` with your GitHub handle
        - [ ] populate `Page` with your selected page from **1. Page Log**
        - [ ] populate `Error Title` with the dropdown that matches the subtitle on the Details pane (see **Resource 2.01.02**)
        - [ ] double check that the `Category` field now matches the title on the Details pane (see **Resource 2.01.01**). If there's not a match, leave a comment with the actual title 
        - [ ] populate the `DOM snippet` by pasting in the highlighted HTML snippet for the Code panel
        - [ ] leave `Steps to Reproduce` empty

- Form-Elements
    - [ ] For all form elements (e.g. text input, password fields, checkboxes), attempt to trigger validation error messages (e.g. missing input, password complexity, incorrect username or password)
      - [ ] In **2. Audit Log**, create and populate row for every Error, Contrast Error, and Alert on the Details pane
        - [ ] populate `Auditor` with your GitHub handle
        - [ ] populate `Page` with your selected page from **1. Page Log**
        - [ ] populate `Error Title` with the dropdown that matches the subtitle on the Details pane (see **Resource 2.01.02**)
        - [ ] double check that the `Category` field now matches the title on the Details pane (see **Resource 2.01.01**). If there's not a match, leave a comment with the actual title
        - [ ] populate the `DOM snippet` by pasting in the highlighted HTML snippet for the Code panel
        - [ ] in a new comment in this issue, write out all steps required to trigger the validation message
          - [ ] populate `Steps to Reproduce` with the link to this comment


- Table-Controls
    - [ ] For all table controls (e.g. dropdown menus, radio buttons, checkboxes, filters) click on the table control to trigger any popups, menu dropdowns, selection styles, etc.
      - [ ] In **2. Audit Log**, create and populate row for every Error, Contrast Error, and Alert on the Details pane
        - [ ] populate `Auditor` with your GitHub handle
        - [ ] populate `Page` with your selected page from **1. Page Log**
        - [ ] populate `Error Title` with the dropdown that matches the subtitle on the Details pane (see **Resource 2.01.02**)
        - [ ] double check that the `Category` field now matches the title on the Details pane (see **Resource 2.01.01**). If there's not a match, leave a comment with the actual title 
        - [ ] populate the `DOM snippet` by pasting in the highlighted HTML snippet for the Code panel
        - [ ] in a new comment in this issue, write out all steps required to trigger the table control elements
          - [ ] populate `Steps to Reproduce` with the link to this comment

 
- Interactive-Elements
    - [ ] For all other interactive elements (e.g. search-boxes, custom input-fields, tooltips, info-buttons) attempt to interact with them so they reveal any informational text, popups, or validation errors
      - [ ] In **2. Audit Log**, create and populate row for every Error, Contrast Error, and Alert on the Details pane
        - [ ] populate `Auditor` with your GitHub handle
        - [ ] populate `Page` with your selected page from **1. Page Log**
        - [ ] populate `Error Title` with the dropdown that matches the subtitle on the Details pane (see **Resource 2.01.02**)
        - [ ] double check that the `Category` field now matches the title on the Details pane (see **Resource 2.01.01**). If there's not a match, leave a comment with the actual title 
        - [ ] populate the `DOM snippet` by pasting in the highlighted HTML snippet for the Code panel
        - [ ] in a new comment in this issue, write out all steps required to trigger the table control elements
          - [ ] populate `Steps to Reproduce` with the link to this comment


### Resources/Instructions

- 1.01 TDM Wiki Pages
  - 1.01.01 [Accessibility Audits](https://github.com/hackforla/tdm-calculator/wiki/Accessibility-Audits)
    - 1.01.02 [How to run the WAVE browser extension](https://github.com/hackforla/tdm-calculator/wiki/Accessibility-Audits#tdm-wave-audits)
  - 1.01.03 https://github.com/hackforla/tdm-calculator/wiki/WAVE-Accessibility-Known-Issues-and-Solutions
- 1.02 _TDM WAVE Accessibility Audit Spreadsheet_
  - 1.02.01 [1. Page Log](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=669073341#gid=669073341)
  - 1.02.02 [2. Audit Log](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0)
  - 1.02.03 [3. Solved Index](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=406857693#gid=406857693)
  - 1.02.04 [4. Unsolved Index](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=1793119266#gid=1793119266)
  - 1.02.05 [5. WAVE Error Index](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=1697403119#gid=1697403119)

#### Audit Instructions

<details><summary>2.01 WAVE Details pane (title and subtitle)</summary>
<p>

> **2.01.01 Title**
> <img height="640" alt="Image" src="https://github.com/user-attachments/assets/954fbfcf-bcff-4ed1-b85d-ed22ee2d460e" />
>
> **2.01.02 Subtitle**
> <img height="640" alt="Image" src="https://github.com/user-attachments/assets/4a57bf7f-fef3-4580-a0c7-0f3d1e5be851" />

</p>
</details> 

<details><summary>2.02 Screenshots of WAVE tool with Register Page, No Interaction</summary>
<p>

## Register Page, No Interaction
> - Page URL: https://tdm-dev.azurewebsites.net/register
> - Steps to Reproduce: n/a
> - Error: Missing form label

**2.02.01 Screenshot (w/o WAVE Annotations)**
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/56a989de-69ec-42d7-a75c-50263519f5a8" />

**2.02.02 Screenshot (w/ WAVE Annotations)**
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/4622dd2e-0594-4879-9b98-775aa19a2d77" />

**2.02.03 Screenshot (click on first Missing form label)**
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/25887728-4022-4ce9-ad71-f9227d53804c" />

**2.02.04 Screenshot (open Code panel, click on first Missing form label)**
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/a50670ce-9c9d-4a98-83a8-9ffc29d66c02" />

**2.02.5 HTML DOM Snippet from firstMissing form label**
```
<input name="firstName" type="text" placeholder="First Name" class="form-control " value="">
```
</p>
</details>

**Comment by ngorokh** [comment]
# Status
- [x] keep
- [ ] delete 

## Title: Lighthouse: Accessibility - Forms

## Labels

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/0b2b0604-cd42-4b17-8873-de573eba1bce" />

## Body
### Overview
In order for your sites form(s) to be usable by visitors using screen readers all the form <input> elements need labels.  There are specific details and exceptions, which can be found in the instructions below.

### Action Items 
*If your site already has forms* review the instructions and document the changes needed to bring your form(s) into WCAG compliance, by commenting on this issue.
*If your site does not have forms* review the instructions and design new forms using the WCAG standards. 

- [ ] Review with product and dev. Update if needed based on feedback
- [ ] Once finalized, add before and after images to the [staging deck on this slide]() 
   - [ ] if no slide has been made, make one in [the staging deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
- [ ] Get Stakeholder sign-off via the stakeholder meeting slide deck.

### Instructions
Deque University 
https://dequeuniversity.com/rules/axe/3.2/label

**Comment by ngorokh** [comment]
# Status
- [x] keep
- [ ] delete 

## Title: Lighthouse Issue: Cross-origin destinations are unsafe
## Labels
<img width="250" alt="Image" src="https://github.com/user-attachments/assets/d8511b9f-4c70-4bbd-a179-fd6751e389ef" />

## Body

### Overview
Links to cross-origin destinations are unsafe both from a security and performance perspective.  

### Action Item
Run [Lighthouse](https://developers.google.com/web/tools/lighthouse/) and then follow the instructions in [cross-origin destinations are unsafe]
(https://developers.google.com/web/tools/lighthouse/audits/noopener) .   
- [ ] Review with product and dev. Update if needed based on feedback
- [ ] Once finalized, add before and after images to the [staging deck on this slide]() 
   - [ ] if no slide has been made, make one in [the staging deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
- [ ] Get Stakeholder sign-off via the stakeholder meeting slide deck.

## Summary of instructions
When using *target=_blank* also adding *rel="noopener"* to the tag ensures that new page runs in a separate process.

**Comment by ngorokh** [comment]
# Status
- [x] keep
- [ ] delete 

## Title: Lighthouse: How To

## Labels

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/9d088483-c6d4-4a86-b322-5497b7d0028d" />

## Body

### Overview
Lighthouse is an open-source, automated tool for improving the quality of web pages. You can run it against any web page, public or requiring authentication. It has audits for performance, accessibility, progressive web apps, and more.  Hack For LA recommends that you run the tests and evaluate what changes you might want to make on your website to improve performance and accessability.

### How To Use
Lighthouse is in the Audits panel of the Chrome DevTools. To run a report:

1. Download Google Chrome for Desktop.
2. In Google Chrome, go to the URL you want to audit. You can audit any URL on the web.
3. Open Chrome DevTools.
4. Click the Audits tab.
5. Click Perform an audit. DevTools shows you a list of audit categories. Leave them all enabled.
6. Click Run audit. After 60 to 90 seconds, Lighthouse gives you a report on the page.

- [ ] Review with product and dev. Update if needed based on feedback
- [ ] Once finalized, add before and after images to the [staging deck on this slide]() 
   - [ ] if no slide has been made, make one in [the staging deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
- [ ] Get Stakeholder sign-off via the stakeholder meeting slide deck.
For more information go to :
https://developers.google.com/web/tools/lighthouse/

### Tip
You will want to re-run lighthouse on any code changes before integrating them into your site.  Sometimes the specific suggestions it makes, do not actually result in improved performance or can actually harm performance.

**Comment by ngorokh** [comment]
# Status
- [ ] keep
- [ ] delete 

## Title: Lighthouse: Image Optimization

## Labels

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/0e750cbe-3dcf-43fa-917f-0cd7503eabfa" />

## Body
### Overview
When you run the lighthouse review it may suggest some specific image optimizations such as choosing another image format and making those changes may or may not improve your sites actual performance.

### Action Items
Run lighthouse on a local version of the website and then apply suggested changes and retest locally before determining if you want to keep the changes. 
- [ ] Review with product and dev. Update if needed based on feedback
- [ ] Once finalized, add before and after images to the [staging deck on this slide]() 
   - [ ] if no slide has been made, make one in [the staging deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
- [ ] Get Stakeholder sign-off via the stakeholder meeting slide deck.

### Instructions/Resources
Google's Tools for Web Developers: [Optimize Images](https://developers.google.com/web/tools/lighthouse/audits/optimize-images)
Read [closed issue #111](https://github.com/hackforla/website/issues/111) from when HackforLA.org did our audit, to see why we decided not to do the image optimization

**Comment by ngorokh** [comment]
# Status
- [ ] keep
- [x] delete 

## Title: Wave Chrome Extension: Accessibility review

## Labels

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/135657da-1287-46d4-9aec-b152ff4f2160" />

## Body
### Overview

#### Action Items
1. add [WAVE chrome extension](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
2. visit site
3. click the extension and review the red flags.
4. Run the same steps for the development site (localhost). Ensure that the chrome extension has the "Allow access to file URLs" enabled.
5. Document all suggested changes in the comments.
- [ ] Review with product and dev. Update if needed based on feedback
- [ ] Once finalized, add before and after images to the [staging deck on this slide]() 
   - [ ] if no slide has been made, make one in [the staging deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
- [ ] Get Stakeholder sign-off via the stakeholder meeting slide deck.

**Comment by ngorokh** [comment]
# Status
- [ ] keep
- [x] delete 
  - [x] keeping some of the content in the accessabiity wiki.  
     - Removing 
        - [aXe](https://www.deque.com/axe/)
        - [SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc)
     - keeping
        - WAVE (already in the accessibility repo)
        - [Lighthouse](https://developers.google.com/web/tools/lighthouse/#devtools) - a google tool
        - potential some of the items from the "bigger list"
## Title: Which accessibility testing tool should you use?

## Labels

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/95a98a92-8838-472d-a0cd-505ddbbdebe9" />

## Body
### Overview
There are more than 100 accessibility testing tools.  Figuring out which ones to use can be a black hole.  For guidance we recommend this article:  [Which accessibility testing tool should you use?](https://medium.com/pulsar/which-accessibility-testing-tool-should-you-use-e5990e6ef0a)

### Summary of Article
The author recommends using the tools in the following order fixing as you go along, since no one tool catches all the relevant issues

[aXe](https://www.deque.com/axe/)
[SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc)
[Tenon](https://tenon.io/)
[WAVE](https://wave.webaim.org/extension/)
[Lighthouse](https://developers.google.com/web/tools/lighthouse/#devtools)

But if you want to test your site with other tools, here is a bigger [list](https://www.w3.org/WAI/ER/tools/)

---
- [ ] Review with product and dev. Update if needed based on feedback
- [ ] Once finalized, add before and after images to the [staging deck on this slide]() 
   - [ ] if no slide has been made, make one in [the staging deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
- [ ] Get Stakeholder sign-off via the stakeholder meeting slide deck.

**Comment by ngorokh** [comment]
# Status
- [ ] keep
- [ ] delete 
## Title: Blank Issue

## Labels

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/f9a80962-2148-441c-88f1-237dbb6bd880" />

## Body
### Overview
REPLACE THIS TEXT -Text here that clearly states the purpose of this issue in 2 sentences or less.

### Action Items
REPLACE THIS TEXT -If this is the beginning of the task this is most likely something to be researched and documented.

REPLACE THIS TEXT -If the issue has already been researched, and the course of action is clear, this will describe the steps.  However, if the steps can be divided into tasks for more than one person, we recommend dividing it up into separate issues, or assigning it as a pair programming task.

- [ ] Review with product and dev. Update if needed based on feedback
- [ ] Once finalized, add before and after images to the slide in the staging deck for this issue (see link under resources)
- [ ] Get Stakeholder sign-off via the stakeholder meeting slide deck.

### Resources/Instructions
REPLACE THIS TEXT -If there is a website which has documentation that helps with this issue provide the link(s) here.

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - [Staging slide, ISSUE NUMBER]
- Release Deck
   - [2025-MM-DD Release deck, slide ISSUE NUMBER]

**Comment by ngorokh** [comment]
# Status
- [ ] keep
- [ ] delete 
## Title: Bug report

## Labels

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/3da3af5a-b422-4fe5-985a-fabf44049be4" />

## Body
#### Describe the bug



#### Steps to Reproduce

1. 

#### Actual Result


<details><summary></summary>


</details>

#### Expected Result


#### Device Configuration
- Device: 
- OS version: 
- Browser: 
- Browser version: 

#### Action Items
- [ ] Fix bug.
- [ ] Review with development lead.
- [ ] Review with product, development, and research teams. Update if needed based on feedback.
- [ ] Once finalized, add a slide to the staging deck (see resources) 
- [ ] Get Stakeholder sign-off via the stakeholder meeting slide deck.

### Resources/Instructions

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - [Staging slide, ISSUE NUMBER]
- Release Deck
   - [2025-MM-DD Release deck, slide ISSUE NUMBER]

**Comment by ngorokh** [comment]
# Status
- [ ] keep
- [ ] delete 
## Title: Feature request

## Labels

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/861acdea-8813-4ae6-a061-440725c80409" />

## Body
### Overview
Please write a user story for this feature suggestion in the following format: As a ______, I want to do X for Y reason and replace this text with it.

### Action Items
- [ ] Define your Feature Suggestion
   - [ ] What is the feature you are suggesting?
   - [ ] Is this feature urgent
      - [ ] if so why?
   - [ ] How long do you expect it take to implement this issue?
   - [ ] Who is needed to implement this feature (Could you implement it yourself?)
      - [ ] Design
      - [ ] Front End
      - [ ] Back End
      - [ ] Database
   - [ ] What problem does this feature solve?
   - [ ] What technologies are needed for this feature?
- [ ] Review with product and dev. Update if needed based on feedback
- [ ] Once finalized, add before and after images to the staging deck (see resources) 
- [ ] Get Stakeholder sign-off via the stakeholder meeting slide deck.

### Resources/Instructions
If there is a website which has documentation that helps with this issue provide the link(s) here:


**Additional context**
Add any other context or screenshots about the feature request here.


#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - [Staging slide, ISSUE NUMBER]
- Release Deck
   - [2025-MM-DD Release deck, slide ISSUE NUMBER]

**Comment by ngorokh** [comment]
# Status
- [ ] keep
- [ ] delete 
## Title: Development Handoff Issue

## Labels

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/9a29697e-07ee-41e4-93cc-3c1ad1f81330" />

## Body

### Overview
Please implement [brief description of design] as described in [design issue number].


### Action Items
- [ ] Assign yourself to the development issue.
- [ ] Move the development issue from the Prioritized Backlog to In Progress.
- [ ] [ADD DEVELOPER ACTION ITEMS]
- [ ] Create a PR, if applicable.
- [ ] Create a new slide (or edit one if it already exists) in staging and add the before and after images
- [ ] Once the PR has been approved, use it to close this issue. If there is no PR is associated with this issue, close this issue after lead signoff.

### Resources/Instructions
- Design details: [link to final comment in GitHub issue]
- Design mockups: [link to Ready for Dev section in the related Figma page]

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - [Staging slide, ISSUE NUMBER]
- Release Deck
   - [2025-MM-DD Release deck, slide ISSUE NUMBER]

**Comment by ngorokh** [comment]
# Status
- [ ] keep
- [ ] delete 
## Title: Design Issue

## Labels

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/03f18981-76e0-4bdf-819e-50b8648bcbd1" />

## Body
### Overview


### Details


### Action Items
- [ ] Assign yourself to this issue.
- [ ] Move the issue from Prioritized Backlog to In Progress.
- [ ] In Figma, move a blank page to the Design section.
- [ ] Change the Figma page's title to have the format "#Issue Number | Title | Assignee Name".
- [ ] Copy contents from the Figma Template page to this issue's Figma page.
- [ ] Include a link to this issue's Figma page in the Figma section below.
- [ ] Use the Figma page to design solutions for this issue.
- Once this issue is ready to be reviewed:
  - [ ] Add a comment tagging the team design lead that [explains the design](https://github.com/hackforla/tdm-calculator/wiki/Design:-Presentations).
  - [ ] Move it to the Visual Review column.
  - [ ] Add the "ready for Design lead-[team]" label specific to your team.
  - [ ] Once you have received feedback from the team lead, implement feedback as necessary.
- [ ] Use [presentation structure](https://github.com/hackforla/tdm-calculator/wiki/Design:-Presentations) to review proposed designs with the design director. Implement feedback as necessary.
- [ ] Use [presentation structure](https://github.com/hackforla/tdm-calculator/wiki/Design:-Presentations) to review proposed designs with the product, development, and research teams. Implement feedback as necessary.
- [ ] Create a new slide (or edit one if it already exists) in staging and add images and appropriate text
- [ ] Receive design approval from stakeholders. Implement feedback as necessary.
- [ ] Add any annotations to the Figma design that would help the developer with implementation.
- [ ] Create a Ready for Dev section around the final designs on the Figma page.
- [ ] Update the [Figma Design System](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=4041-12655&t=54f7L4Eubz7ppygs-1) with changes resulting from the issue's final design, if applicable.
- [ ] Update the [Style Guide Presentation](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/), if applicable.
- [ ] Add a comment to the issue that references the final design and adds any annotations that would help the developer with implementation.
- [ ] Create a [development handoff issue](https://github.com/hackforla/tdm-calculator/issues/new?template=development-handoff-issue.md) that references the comment.
- [ ] Move the Figma page to the Development section of the Figma file.
- [ ] Add the development issue's number in parentheses at the beginning of the title of the Figma page.
- [ ] Include the development handoff issue number in the GitHub section below.
- [ ] Reassign any previous assignees to this issue.
- [ ] Close this issue.

### Resources

#### GitHub
- 

#### Figma
- 

#### Dev Site
-

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - [Staging slide, ISSUE NUMBER]
- Release Deck
   - [2025-MM-DD Release deck, slide ISSUE NUMBER]

**Comment by ngorokh** [comment]
# Status
- [ ] keep
- [ ] delete 
## Title: Content: FAQ database: Add content

## Labels

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/5e7c6f02-1b2f-4857-98f3-a34a51533487" />

## Body
### Overview
We need to add content to the FAQ that explains to people how [Replace with NAME OF ITEM] works.

### Details
The below text provides a general outline for the question and answer. Use it as the basis for writing new content for this FAQ.

#### Question (draft)
[Replace with PROPOSED QUESTION]

#### Answer (draft)
[Replace with PROPOSED ANSWER OR OTHER DETAILS THAT WILL HELP WRITE DRAFT]

### Action Items
- [ ] Assign yourself to this issue.
- [ ] Move the issue from Prioritized Backlog to In Progress.
- [ ] Make sure there is a slide linked under resources.  It not, make one in the staging deck
- [ ] Create a document in the [TDM Google Drive, FAQ content folder](https://drive.google.com/drive/folders/1W3Pk6TTq1lVYreTHipc5H-jCvyvByrnS), called FAQ Question: [Replace with NAME OF ITEM  - ISSUE #]
- [ ] Write a draft of the category, question, and answer that would appear on the FAQ page
- [ ] Review proposed content with the design lead. Implement feedback as necessary.
- [ ] Review proposed content with the design director. Implement feedback as necessary.
- [ ] Review proposed content with the product, development, and research teams. Implement feedback as necessary.
- Receive content approval from stakeholders by 
   - [ ] adding a link to the document onto the related slide (see resources below for staging slide).
   - [ ] add a comment on the slide with the text
      ```
      add to stakeholder release deck
      ```
- [ ] Implement feedback as necessary.
- [ ] Once approved, sign in as an admin and Update the FAQ
- [ ] Create a new slide (or edit one if it already exists) in staging and add a screenshot of the Q and A being added
   - [ ] add a comment on the slide with the text
      ```
      add to stakeholder release deck
      ```
- [ ] Reassign any previous assignees to this issue.
- [ ] Close this issue.

### Resources

#### GitHub
- Updating Terms and Conditions: #2253 

#### Google Drive
- [TDM Google Drive, FAQ content folder](https://drive.google.com/drive/folders/1W3Pk6TTq1lVYreTHipc5H-jCvyvByrnS)

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - [Staging slide, ISSUE NUMBER]
- Release Deck
   - [2025-MM-DD Release deck, slide ISSUE NUMBER]

**Comment by ngorokh** [comment]
# Status
- [ ] keep
- [ ] delete 
## Title: Content: FAQ database: Revise content

## Labels

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/0ebe4873-3382-46a2-9dd5-fb33029cea91" />

## Body
### Overview
We need to revise answer content in the FAQ for the question about [Replace with QUESTION SUMMARY].

### Details
The below text provides a general outline for the question and answer. Use it as the basis for writing new content for this FAQ.

#### Category Current
[Replace with CURRENT CATEGORY]

#### Question Current
[Replace with QUESTION TEXT]

#### Answer Current
[Replace with ANSWER TEXT]

#### Answer Draft Notes
[Replace with PROPOSED ANSWER OR OTHER DETAILS THAT WILL HELP WRITE DRAFT, INCLUDING REASON FOR CHANGE]

### Action Items
- [ ] Assign yourself to this issue.
- [ ] Move the issue from Prioritized Backlog to In Progress.
- [ ] Make sure there is a slide linked under resources.  It not, make one in the staging deck
- [ ] Create a document in the [TDM Google Drive, FAQ content folder](https://drive.google.com/drive/folders/1W3Pk6TTq1lVYreTHipc5H-jCvyvByrnS), called FAQ Question: [Replace with NAME OF ITEM - ISSUE #]
- [ ] Add the Current category, question, and answer that appears on the FAQ page
- [ ] Add the proposed changes 
- [ ] Review proposed content with the design lead. Implement feedback as necessary.
- [ ] Review proposed content with the design director. Implement feedback as necessary.
- [ ] Review proposed content with the product, development, and research teams. Implement feedback as necessary.
- Receive content approval from stakeholders or ask them to help with content by 
   - [ ] adding a link to the document onto the related slide (see resources below for staging slide).
   - [ ] add a comment on the slide with the text
      ```
      add to stakeholder release deck
      ```
- [ ] Implement feedback as necessary.
- [ ] Once approved, sign in as an admin and Update the FAQ
- [ ] Create a new slide (or edit one if it already exists) in staging and add a screenshot of the Q and A being revised
   - [ ] add a comment on the slide with the text
      ```
      add to stakeholder release deck
      ```
- [ ] Reassign any previous assignees to this issue.
- [ ] Close this issue.

### Resources

#### GitHub
- Updating Terms and Conditions: #2253 

#### Google Drive
- [TDM Google Drive, FAQ content folder](https://drive.google.com/drive/folders/1W3Pk6TTq1lVYreTHipc5H-jCvyvByrnS)

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - [Staging slide, ISSUE NUMBER]
- Release Deck
   - [2025-MM-DD Release deck, slide ISSUE NUMBER]
